manifests/params.pp

```diff
@@ -225,4 +225,7 @@
       $plugin_prefix = 'smart_proxy_'
     }
   }
+
+  $puppet_use_cache = undef
+  $puppet_cache_location = '/var/lib/smart-proxy/cache'
```

Comment:
I just copied it from the puppet.yml.example. Maybe `/var/cache/foreman-proxy` would be a better place, but we should standardize on it.